test/testDataAndAlgorthm.js

```diff
@@ -0,0 +1,32 @@
+"use strict";
+
+var assert = require('assert');
+var arrayMethods = require('../lib/index.js');
+
+describe('test our array functions', function() {
+    it('should pop last index value of array', function() {
+        var testArray = [1, 2, 3];
+        assert.equal(arrayMethods.myPop(testArray), 3, 'did not pass');
```

Comment:
If you add a message to `assert`, it's meant to provide additional details or disambiguate from other asserts in the test method. `did not pass` doesn't add anything.